core/src/main/java/org/apache/accumulo/core/client/impl/AccumuloClientImpl.java

```diff
@@ -53,11 +53,10 @@
 
 import edu.umd.cs.findbugs.annotations.SuppressFBWarnings;
 
-public class AccumuloClientImpl extends org.apache.accumulo.core.client.Connector
-    implements AccumuloClient {
+public class AccumuloClientImpl implements AccumuloClient {
```

Comment:
Does this code still work with this change?
```java
  /**
   * Creates a Connector from an AccumuloClient.
   *
   * @since 2.0
   */
  public static Connector from(AccumuloClient client) {
    return (Connector) client;
  }
```

Reply:
No, but why would we need it to?

Reply:
Oh wait, I fixed this.